gslib/tests/test_mb.py

```diff
@@ -133,3 +133,53 @@ def test_create_with_pap_invalid_arg(self):
                             expected_status=1,
                             return_stderr=True)
     self.assertRegexpMatches(stderr, r'invalid_arg is not a valid value')
+
+  @SkipForXML('Rpo flag only works for GCS JSON API')
+  def test_create_with_rpo_async_turbo(self):
+    bucket_name = self.MakeTempName('bucket')
+    bucket_uri = boto.storage_uri('gs://%s' % (bucket_name.lower()),
+                                  suppress_consec_slashes=False)
+    self.RunGsUtil(
+        ['mb', '-l', 'nam4', '--rpo', 'ASYNC_TURBO',
```

Comment:
Maybe add a comment explaining the bucket location `nam4` b/c it seems unrelated to RPO

Reply:
Done.